### PR TITLE
Box shadows

### PR DESCRIPTION
**Pull request by AdrianJar**

### CODE REVIEW

File: src/common/theme.js
Comment (KatarzynaZaloba) [inline src/common/theme.js:1]: How can I say more?
It looks really nice now!
image

File: src/feature/People/PeopleList/List/styled.js
Comment (KatarzynaZaloba) [inline src/feature/People/PeopleList/List/styled.js:1]: Thanks for adding this to the Peoplelist!
Now the long names of the actors are not outside the tile! 😄 
image

**Review by KatarzynaZaloba — APPROVED** [review]
I totally approve!
It's nice to see that our app is every that better 😃

**Review by Marianna-Weychan — APPROVED** [review]
oh that was a good move! Our app is starting to look better and better!
Great that you added this CSS to your theme, definitely aprove!

**Pull request merged**